products/magento.md

```diff
@@ -4,62 +4,79 @@ category: server-app
 iconSlug: magento
 permalink: /magento
```

Comment:
Since we don't have a dedicated page for now, perhaps a redirect makes sense.

```suggestion
permalink: /magento
alternate_urls:
-   /adobe-commerce
```

Reply:
I agree.